biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java

```diff
@@ -1061,12 +1066,19 @@ public int compareTo(PropertyKey o) {
 		 * @param keys sorted keys defined by {@link #compareTo(PropertyKey)}
```

Comment:
@pkriens I see you are sorting in the method now.
Maybe then the "sorted" for `keys` should be removed from the javadoc